packages/block-editor/src/components/block-bindings-toolbar-indicator/style.scss

```diff
@@ -1,14 +1,16 @@
 .block-editor-block-bindings-toolbar-indicator {
 	display: inline-flex;
 	align-items: center;
-	height: 48px;
+	justify-content: center;
 	padding: 6px;
-
-	svg g {
-		stroke: var(--wp-bound-block-color);
-		fill: transparent;
-		stroke-width: 1.5;
-		stroke-linecap: round;
-		stroke-linejoin: round;
+	svg {
```

Comment:
We can now remove those properties because of the updates to the <path> of the SVG.